src/importer/reducer.ts

```diff
@@ -30,7 +33,10 @@ function recalculateCalculatedColumns(
   return row;
 }
 
-function buildInitialState(sheetDefinitions: SheetDefinition[]): ImporterState {
+function buildInitialState(
+  sheetDefinitions: SheetDefinition[],
+  indexDBConfig: IndexDBConfig = { enabled: false }
```

Comment:
1. I think the default should be defined in ImporterBody
2. Shouldn't it be turned on by default?

Reply:
1. I'll update
2. I'm not sure if we should automatically save to their indexDB. A single device such as a mac could be used by multiple people. Maybe data that is upload using one person's credentials would be seen by someone else using the same device. This feature is probably better off something that is manually turned on instead.

Reply:
2. I think that makes sense.  How would that work for labconnect though? Would integrator need to add a customKey and store userId in it so that different users would have different stores?

Reply:
@czhu12 Idk what your vision is for this, but would we just leave it up to the user to figure out the logic?

Reply:
If we leave the problem to integrator we should definitely describe that in the docs